features/definitions/resolvers-steps.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const should = require('should'),
+  { When, Then } = require('cucumber');
+
+const baseRequest = {
+  controller: 'kuzzle-plugin-graphql/graphql',
+  action: 'endpoint',
+  refresh: 'wait_for',
+};
+
+When(
+  'I successfully call the GraphQL endpoint with the following body:',
+  async function (data) {
+    try {
+      const response = await this.sdk.query({
+        ...baseRequest,
+        body: JSON.parse(data),
+      });
+
+      this.props.result = response.result;
+    } catch (error) {
```

Comment:
In our coding standard we add a newline before the `catch` block to have a better visual separation between both blocks